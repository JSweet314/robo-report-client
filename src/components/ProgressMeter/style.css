.complaint-progress {
  width: 50%;
  margin: 0rem auto 1rem auto;
}

.complaint-progress meter {
  width: 100%;
  height: 2rem;
}