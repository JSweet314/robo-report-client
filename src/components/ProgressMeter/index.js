import React from 'react';
import PropTypes from 'prop-types';
import './style.css';

const ProgressMeter = ({ blockIndex, min, max }) => {
  return (
    <div className='complaint-progress'>
      <meter value={blockIndex} min={min} max={max}/>
    </div>
  );
};

ProgressMeter.propTypes = {
  blockIndex: PropTypes.number.isRequired,
  min: PropTypes.number.isRequired,
  max: PropTypes.number.isRequired
};

export default ProgressMeter;