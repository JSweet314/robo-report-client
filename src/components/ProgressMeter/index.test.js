import React from 'react';
import { shallow } from 'enzyme';
import ProgressMeter from './index';

describe('PrgressMeter', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<ProgressMeter blockIndex={0} min={-1} max={5}/>);
  });

  it('should match a snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});