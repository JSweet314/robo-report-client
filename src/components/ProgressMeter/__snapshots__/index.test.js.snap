// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrgressMeter should match a snapshot 1`] = `
<div
  className="complaint-progress"
>
  <meter
    max={5}
    min={-1}
    value={0}
  />
</div>
`;
